gadget/validate.go

```diff
@@ -208,6 +209,39 @@ func ensureSystemSaveRuleConsistency(state *validationState) error {
 
 // content validation
 
+// XXX: needs to match kernel/kernel.go
+var validKernelAssetName = regexp.MustCompile("^[a-zA-Z0-9][a-zA-Z0-9-]*$")
```

Comment:
shouldn't we import it from there then?  or a Validate* functions?

Reply:
done